src/applications/coronavirus-vaccination-expansion/config/form.js

```diff
@@ -38,6 +38,13 @@ const formConfig = {
     reviewPageTitle: 'Review your information',
     appType: 'form',
   },
+  additionalRoutes: [
+    {
+      path: 'confirmation',
+      component: ConfirmationPage,
+      pageKey: 'confirmation',
+    },
+  ],
```

Comment:
I don't know if this will work like we want, but I'm doing it because if I set the `confirmation` property then we get the additional form title on top of the confirmation page, and I'm not sure how to remove it. This avoids that.

![image](https://user-images.githubusercontent.com/2008881/112772042-8c5a2c00-8fe3-11eb-8fbe-f75f35b23362.png)